promtimer/promtimer.py

```diff
@@ -252,7 +285,63 @@ def main():
     grafana_port = args.grafana_port
     prometheus_base_port = grafana_port + 1
 
-    if not args.cluster:
+    cbbackupmgr_stats_mode = False
+    buckets = None
+
+    if args.stats_archive_path:
+        cbbackupmgr_stats_mode = True
+
+        archive_path = args.stats_archive_path
+        if zipfile.is_zipfile(args.stats_archive_path):
+            # Using tempfile.gettempdir() sometimes causes the Prometheus server to take a long time to start up
+            archive_path = os.path.join(tempfile.gettempdir(), 'stats-archive')
+            if os.path.exists(archive_path):
+                shutil.rmtree(archive_path)
+            os.mkdir(archive_path)
```

Comment:
`tempfile` has a useful class to create a temporary directory. I think I would prefer using that as it should guarantee you get a directory that doesn't exist. Also applies a few lines below

https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory